src/umpire/util/AllocationMap.cpp

```diff
@@ -265,20 +268,28 @@ bool AllocationMap::contains(void* ptr) const
 
 void AllocationMap::clear()
 {
+  // Aquire lock
+  std::lock_guard<std::mutex> lock(m_mutex);
+
   UMPIRE_LOG(Debug, "Clearing");
 
-  UMPIRE_LOCK;
   m_map.clear();
   m_size = 0;
-  UMPIRE_UNLOCK;
 }
 
-size_t AllocationMap::size() const { return m_size; }
+size_t
+AllocationMap::size() const
+{
+  return m_size;
+}
 
 void
 AllocationMap::print(const std::function<bool (const AllocationRecord&)>&& pred,
                      std::ostream& os) const
 {
+  // Aquire lock
```

Comment:
This is pedantic, but I think these comments above every `lock_guard` are redundant.

Reply:
ok! I'll remove them

Reply:
thanks :)